app/models/assignment_override.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import (
+    Column, Sequence, ForeignKey,
+    Integer, Interval, DateTime
+)
+from sqlalchemy.schema import UniqueConstraint
+from sqlalchemy.orm import relationship, backref
+from app.database import Base
+
+class AssignmentOverrideModel(Base):
+    __tablename__ = "assignment_override"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
I'd like to either change our system to use LMS resource ids as our primary keys across every LMS resource table or move away from this practice and store the LMS identifier separate such as `lms_id` to be more clear.